run-build-functions.sh

```diff
@@ -281,6 +281,11 @@ install_dependencies() {
     fi
   fi
 
+  if [ "$NODE_ENV" == "production" ]
```

Comment:
Should we put this right before `run_npm`/`run_yarn` instead? Like this, we will not print this when the site has no `package.json`. Granted though that having a `NODE_ENV` environment variable but no `package.json` should be unlikely.